src/components/Breadcrumbs/Breadcrumb.js

```diff
@@ -1,37 +1,36 @@
-import React from "react";
+import React, { useMemo } from "react";
 import PropTypes from "prop-types";
-import { Link } from "gatsby";
-import { handlePath } from "../../helpers/handlers";
+import { Breadcrumb } from "gatsby-plugin-breadcrumb";
 
 import "./Breadcrumb.css";
 
-const Breadcrumb = ({ crumbs }) => {
+const sanitizeCrumbs = (crumbs) => {
+  // Filtering not required crumbs to prevent them from rendering.
+  const sanitizedCrumbs = crumbs.filter((crumb) => {
+    return crumb.pathname !== `/ua/`;
```

Comment:
What's this?

Reply:
Removing automatically generated crumb with pathname /ua/ to prevent it from rendering.

Reply:
Issue resolved by adding hiddenCrumbs prop instead of modifying current crumbs array.